Sources/Commands/APIDigester.swift

```diff
@@ -116,77 +133,137 @@ struct APIDigesterBaselineDumper {
             stdoutStream: stdoutStream
         )
 
-        // FIXME: Need a way to register this build operation with the interrupt handler.
-
         try buildOp.build()
 
         // Dump the SDK JSON.
-        try apiDigesterTool.dumpSDKJSON(
-            at: sdkJSON,
-            modules: graph.apiDigesterModules,
-            additionalArgs: buildOp.buildPlan!.createAPIToolCommonArgs(includeLibrarySearchPaths: false)
-        )
+        try localFileSystem.createDirectory(baselineDir, recursive: true)
+        let group = DispatchGroup()
+        let errors = ThreadSafeArrayStore<Swift.Error>()
+        for module in modulesToDiff {
+            DispatchQueue.sharedConcurrent.async(group: group) {
```

Comment:
Thanks for adding this. Could you add support to control concurrency as well (DispatchSemaphore)? Will need to add a "-j" input option to the cmdline. This can be a separate PR though.

Reply:
I ended up making this change in this PR since the command already accepted `-j` to control the parallelism of the build operation. Now both stages should respect the option consistently

Reply:
Thanks for adding semaphore.